repos.yaml

```diff
@@ -33,3 +33,13 @@ repositories:
   maintainers:
     - email: rmocius@gmail.com
       name: rimusz
+- name: buildkite
+  url: https://buildkite.github.io/charts
+  maintainers:
+    - email: rmocius@gmail.com
+      name: rimusz
+- name: keel
+  url: https://charts.keel.sh 
+  maintainers:
+    - email: karolis.rusenas@gmail.com
+      name: rusenask
```

Comment:
Can we get @rusenask to approve of this in this PR? We do have certain requirements ([see docs](https://github.com/helm/hub/blob/master/Repositories.md)) and his email address will be publicly stored.